plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/exporter/OtlpGrpcSpanExporterProvider.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing.exporter;
+
+import org.opensearch.common.settings.Settings;
+import org.opensearch.telemetry.OTelTelemetrySettings;
+
+import java.util.Collection;
+
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+/**
+* OtlpGrpcSpanExporterProvider class creates an instance of OtlpGrpcSpanExporter
+*/
+public final class OtlpGrpcSpanExporterProvider implements SpanExporter {
```

Comment:
@Shephalimittal @shwetathareja this is not the extensible way to deal with OTEL span exporters: we basically need to use wrappers for each and every one of them. I understand that we would like to have some initial implementation that work but we need also to think a bit into the future so we won't be asking people to go over painful settings migrations.

I would susggest:
  - use OtlpGrpcSpanExporter directly 
  - use (for now) `telemetry.otel.tracer.exporter.otlphrpcspanexporter.endpoint` setting (instead of generic one `telemetry.otel.tracer.exporter.endpoint`, not all exporters deal with that)
  - `OTelSpanExporterFactory` should recognize `<SpanExporterCass>::builder` class 
  - `OTelSpanExporterFactory` should extract   `telemetry.otel.tracer.exporter.<SpanExporterCass>.*` settings and call respective builder setters

This is not 100% bullet proof but would later prevent at least setting changes.

Reply:
The `telemetry.otel.tracer.exporter.otlpgrpcspanexporter.endpoint` is specific to `OtlpGrpcSpanExporter` and no other exporters. Jaeger or Zipkin ones would be using HTTP endpoints that would be specified for their respective exporters only. Also, there are file based exporters or Kafka based one, for them the endpoint is not relevant.

Reply:
@reta I have revised the PR to remove support for configurable settings as of now and instead of checking create() , added check for getDefault() method in the exporter (with default settings). Also instead of creating provider, called  OTLPGrpcSpanExporter directly. This would provide users to visualise spans using OTLPGrpcSpanExporter.

For providing support for configurable settings, I will soon raise a RFC for discussion on use of adding auto-configurable sdk or any alternate solutions to read the configurable settings in custom exporters, samplers etc.
Let me know, in case of any concerns?